src/main/java/org/folio/inventory/services/MoveItemIntoStatusService.java

```diff
@@ -57,14 +61,38 @@ public MoveItemIntoStatusService(ItemCollection itemCollection, Clients clients)
       .thenCompose(itemCollection::update);
   }
 
+  public CompletableFuture<Item> processMarkItem(WebContext context,ItemStatusName statusName) {
+    final String itemId = context.getStringParameter("id", null);
+    AbstractTargetValidator targetValidator = validator.getValidator(statusName);
+
+    return itemCollection.findById(itemId)
+      .thenCompose(ItemsValidator::refuseWhenItemNotFound)
+      .thenCompose(item -> targetValidator.itemHasAllowedStatusToMark(item))
+      .thenCompose(this::updateRequestStatusIfRequired)
+      .thenApply(item -> item.changeStatus(statusName))
+      .thenCompose(itemCollection::update);
+  }
+
+//  public CompletableFuture<Item> processMarkItemInProcess(WebContext context) {
+//    final String itemId = context.getStringParameter("id", null);
+//
+//    return itemCollection.findById(itemId)
+//      .thenCompose(ItemsValidator::refuseWhenItemNotFound)
+//      .thenCompose(MarkAsInProcessValidators::itemHasAllowedStatusToMarkAsInProcess)
+//      .thenCompose(this::updateRequestStatusIfRequired)
+//      .thenApply(item -> item.changeStatus(IN_PROCESS))
+//      .thenCompose(itemCollection::update);
+//  }
```

Comment:
Please remove this commented out code. If it is needed in future it can be retrieved from version control.